### ISSUE TITLE
Cannot compile from sources

### ISSUE THREAD

**Issue by scaramacai** [KJ7LNW/xnec2c#3]
I tried with both the source from master and with the tar.gz of the 4.3.4 version and I got the following error:
Preparing the xnec2c build system...please wait

Found GNU Autoconf version 2.69
Found GNU Automake version 1.16.1
Found GNU Libtool version 2.4.6

Automatically preparing build ... Warning: autoreconf failed
Attempting to run the preparation steps individually

Preparing build ... configure.ac:79: error: required file './config.rpath' not found
configure.ac:79: error: required file './ABOUT-NLS' not found
ERROR: automake failed

I'm on Ubuntu 20.04

Thank you

Gigi

**Comment by ewheelerinc** [comment]
I'm having trouble reproducing the exact error on my Ubuntu 20.04 system, but I have lots of extra packages installed.

Try this:

```sh
apt-get update
apt-get install gettext gettext-devel autopoint 
```
If that doesn't work, try `apt-get install gnulib`

If it still doesn't work, then send me the output of `dpkg -l` so I can troubleshoot.

I'll be out of town until after Christmas so I can continue to troubleshoot with you when I return.

**Comment by ewheelerinc** [comment]
Actually it looks like gettext-devel isn't a thing, exclude that.

**Comment by ewheelerinc** [comment]
I updated the documentation, added some checks in autogen.sh and pushed version 4.3.5.  See the top of this page for detail: https://www.xnec2c.org/

Hopefully that will help others if they have a similar problem.  If this worked for you then please close the issue.

**Comment by scaramacai** [comment]
It looked like I was missing autopoint. The new version helped to fix this and now xnec2c 4.3.5 is compiling and running.
